src/plotman/job.py

```diff
@@ -412,7 +412,23 @@ def status_str_long(self) -> str:
             dst = self.dstdir,
             logfile = self.logfile
             )
-    
+
+    def print_logs(self, follow=None):
```

Comment:
I think this might be enough to satisfy the type hints checking.

```suggestion
    def print_logs(self, follow: bool = False) -> None:
```